### PR TITLE
Added a help page for iOS add-to-app.

### PR DESCRIPTION
**Pull request by gaaclarke**
Brought over info from: https://github.com/flutter/flutter/wiki/Add-Flutter-to-existing-apps
* Cleaned up markdown errors
* Added intro section
* Removed references to Android

### CODE REVIEW

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+  end
+  target 'MyAppTests' do
+    install_all_flutter_pods(flutter_application_path)
+  end
+```
+
+3. Run `pod install`.
+
+Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
+you need to run `flutter packages get` from `some/path/my_flutter` to refresh the list
```
Comment (jmagman) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: I edited this text yesterday, `flutter packages get` was rebranded to `flutter pub get`. https://github.com/flutter/flutter/pull/33041
https://github.com/flutter/flutter/wiki/Add-Flutter-to-existing-apps/_history

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+example that makes sense for the blank screen of the host app generated
+by Xcode 10.0.
+
+First declare your app delegate to be a subclass of `FlutterAppDelegate`. Then define a FlutterEngine property, which help you to register a plugin without a FlutterViewController instance.
+
+In `AppDelegate.h`:
+
+```objective-c
+#import <UIKit/UIKit.h>
+#import <Flutter/Flutter.h>
```
Comment (jmagman) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: I changed all the `#import`s (and one include!) in the wiki yesterday to module imports or project imports.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+are embedded in your project.
+
+You should now be able to build the project using `⌘B`.
+
+### Under the hood
+
+If you have some reason to do this manually or debug why these steps aren't working, here's what's going on under the hood:
+
+1. `Flutter.framework` (the Engine library) is getting embedded into your app for you.  This has to match up with the release type
+(debug/profile/release) as well as the architecture for your app (arm*, i386, x86_64, etc.).  CocoaPods pulls this in as a vendored
```
Comment (jmagman) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Yesterday I removed the reference to i386 since Flutter.framework does not contain that slice.
https://github.com/flutter/flutter/wiki/Add-Flutter-to-existing-apps#under-the-hood

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+vary on the pod dependencies of each plugin.
+4. A build script is added to the Podfile targets that call `install_all_flutter_pods` to ensure that the binaries you build stay up to date
+with the Dart code that's actually in the folder.  It also uses your Xcode build configuration (Debug, Profile, Release) to embed the matching
+release type of `Flutter.framework` and `App.framework`.
+
+## Write code to use FlutterViewController from your host app
+
+The proper place to do this will be specific to your host app. Here is an
+example that makes sense for the blank screen of the host app generated
+by Xcode 10.0.
```
Comment (jmagman) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Can you remove the "10.0" version call-out?  We're about to up the required version to 10.2 and I already updated the Swift syntax so it's probably not what 10.0 generated anymore.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+@property (strong, nonatomic) UIWindow *window;
+@property (nonatomic,strong) FlutterEngine *flutterEngine;
+@end
+```
+
+The implementation should mostly just delegate to a `FlutterPluginAppLifeCycleDelegate`:
+
+```objective-c
+@implementation AppDelegate
+{
```
Comment (jmagman) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: We probably shouldn't encourage an ivar like this.
```objective-c
@interface AppDelegate ()
@property (readonly) FlutterPluginAppLifeCycleDelegate* lifeCycleDelegate;
@end
```
...
```objective-c
return [self.lifeCycleDelegate application:application didFinishLaunchingWithOptions:launchOptions];
```

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 4 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 2 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: I changed the one include.  I did a search on github and `#import` is more popular than `@import` (~500m vs ~400m).  I think we might want to keep it with `#import`.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 5 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Do you have a style guide to reference?  That isn't specified in the google objective-c style guide.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 6]
```diff
@@ -0,0 +1,493 @@
...
+  load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
+```
+
+2. For each Xcode [target](https://guides.cocoapods.org/syntax/podfile.html#target) that needs to embed Flutter, call `install_all_flutter_pods(flutter_application_path)`.
+
+```ruby
+  target 'MyApp' do
+    install_all_flutter_pods(flutter_application_path)
+  end
+  target 'MyAppTests' do
```
[the remark this answers is not in the document]
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Can you just update it after you've tested it?  or let me know and I can change it quickly in another PR.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 7]
```diff
@@ -0,0 +1,487 @@
...
+short-title: Add a Flutter screen
+description: Learn how to add a single Flutter screen to your existing iOS app.
+---
+
+This guide describes how to add a single Flutter screen to an existing iOS app.
+
+## Integration Steps
+
+ 1. [Create Flutter Module](#create-a-flutter-module) - This creates the Flutter project which will be hosted in the existing app.
+ 1. [Add Module Dependency in Existing App](#make-the-host-app-depend-on-the-flutter-module) - This sets up XCode to include your Flutter project into the build of your existing app.
```
Comment (blasten) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: nit: `2.` and below `3.`

Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Markdown automatically changes the numbers, its a lint error not to use that.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 2 above)
Comment (jmagman) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: I think it should be a module import using the `@import` directive. It's a relatively new directive as of clang 10, I believe, so it's not surprising it's not used much in GitHub.

`#import <Framework/Framework.h>` will automatically fall back to a module import and the module cache assuming `Framework.h` it's the umbrella header, so it's essentially the same as `@import Framework;`.  https://clang.llvm.org/docs/Modules.html#includes-as-imports

However, in `<FlutterPluginRegistrant/GeneratedPluginRegistrant.h>` `GeneratedPluginRegistrant.h` is NOT the umbrella header, so it's not equivalent.  FlutterPluginRegistrant is correctly implemented as a module.  CocoaPods generates a `FlutterPluginRegistrant-umbrella.h` but that's kind of a gross implementation detail that I wouldn't want to expose.

TL;DR we should use the suggested modern language features in our docs.

Comment (jmagman) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: (and there's definitely no reason to use `#include`s instead of `#imports`)

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 7 above)
Comment (blasten) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: interesting. I wonder why. I didn't get that error on https://github.com/flutter/website/pull/3080/files#diff-ec1d6c46733f84146f59353836e97271R26

Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Probably because of the 50 things the webpage installs to run the webpage, a markdown linter isn't one of them haha.  Leading with `1.` makes it easier to reorder and insert new items.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 2 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: For question on style I defer to the google objective-c style guide.  http://google.github.io/styleguide/objcguide.html#use-umbrella-headers-for-system-frameworks

It makes no recommendation between the styles, just that you use the umbrella header (implementation wise they should behave the same).  `<FlutterPluginRegistrant/GeneratedPluginRegistrant.h>` is not an umbrella import like you said, why didn't we make the umbrella header `<FlutterPluginRegistrant/FlutterPluginRegistrant.h>`?  Seems like we should have done that.  

> (and there's definitely no reason to use `#include`s instead of `#imports`)

Yep, I removed it.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 5 above)
Comment (jmagman) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: It's mentioned in the "You Can Define Instance Variables without Properties" section.
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html#//apple_ref/doc/uid/TP40011210-CH5-SW23

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jmagman) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Let's just remove the MyAppTests totally, I'd rather omit something than say something wrong.  I can put put out a PR after I test it.

```ruby
  target 'MyApp' do
    install_all_flutter_pods(flutter_application_path)
  end
```

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 2 above)
Comment (jmagman) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: > `<FlutterPluginRegistrant/GeneratedPluginRegistrant.h>` is not an umbrella import like you said, why didn't we make the umbrella header `<FlutterPluginRegistrant/FlutterPluginRegistrant.h>`? Seems like we should have done that.

Flutter plugins were not designed with modules in mind.  ☹️  I'm working on that now. 
https://github.com/flutter/flutter/pull/41828
https://github.com/flutter/flutter/pull/42204

Apple's docs are so bad for this, but I found an indexed copy of WWDC 2014 "What's new in LLVM" that gives guidelines for imports:
https://docs.huihoo.com/apple/wwdc/2014/417_whats_new_in_llvm.pdf

> Use `@import` when importing your framework in another target
Use `#import` in the implementation to textually include the framework headers

I think one of the reasons `@import` became preferred is because it proves the framework has their module set up correctly, and therefore is importable to Swift.  `#import <Framework/Framework.h>` doesn't prove the same thing, just that that header is public in a framework since the module import is implicit.  

See my sadness trying to import our Objective-C plugins without modules as libraries here:
https://github.com/flutter/flutter/issues/41007

Comment (jmagman) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Plus the Google Obj-C guide you linked to also says `@import` directives are "GOOD" but doesn't really give guidelines for when to use them, so it's not against that style guide.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 8]
```diff
@@ -0,0 +1,487 @@
...
+ 1. [Add FlutterViewController to Existing App](#write-code-to-use-flutterviewcontroller-from-your-host-app) - This steps introduces the code that will conjure the instance of Flutter into your existing app.
+ 1. (optional) [Attach Flutter Tool](#building-and-running-your-app) - This step allows you to attach to Flutter's debugger and enables you do use hot reload / restart.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: =>
```
1. (Optional) [Attach Flutter tool](#xx)<br>
Allows you to attach to Flutter's debugger and enables hot reload and hot restart.
```

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 9]
```diff
@@ -0,0 +1,487 @@
...
+
+## Create a Flutter module
+
+Flutter projects created using `flutter create xxx` include very simple host
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: `flutter create xxx` => `flutter create <myapp>`

Also, delete "very".

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 10]
```diff
@@ -0,0 +1,487 @@
...
+apps for your Flutter/Dart code (a single-ViewController iOS host). You can
+modify these host apps to suit your needs and build from there.
+
+But if you're starting off with an *existing* host app for either platform,
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Delete "But".

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 11]
```diff
@@ -0,0 +1,487 @@
...
+you'll likely want to include your Flutter project in that app as some form of
+library instead.
+
+This is what the Flutter module template provides. Executing
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: "This is what the Flutter module template provides." What do you mean? Do you mean that it creates the second option above?

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 12]
```diff
@@ -0,0 +1,487 @@
...
+`flutter create -t module xxx` produces a Flutter project containing a CocoaPods
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Do we need both sentences? Maybe the previous sentence can be deleted?

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 13]
```diff
@@ -0,0 +1,487 @@
...
+pod designed for consumption by your existing host app.
+
+Let's assume you have an existing iOS app at `some/path/MyApp`, and that you
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Delete "let's".  => 
```
Assume that you have an existing iOS app at `some/path/MyApp`,
and you want your Flutter project as a sibling:
```

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 14]
```diff
@@ -0,0 +1,487 @@
...
+want your Flutter project as a sibling:
+
+```bash
+cd some/path/
+flutter create -t module my_flutter
+```
+
+This creates a `some/path/my_flutter/` Flutter module project with some Dart
+code to get you started and a `.ios/` hidden subfolder that wraps up the
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]:
```
to get you started and => to get you started, and a `.ios/` hidden subfolder.
The ios folder wraps the module containing some CocoaPods and a helper
Ruby script.
```

The original sentence was quite a mouthful, so I'm trying to break it up a bit.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 15]
```diff
@@ -0,0 +1,487 @@
...
+module project that contains some Cocoapods and a helper Ruby script.
+
+## Make the host app depend on the Flutter module
+
+The description below assumes that your existing iOS app has a structure similar
+to what you get by asking Xcode version 10.0 to generate a new "Single View App"
+project using Objective-C. If your existing app has a different folder structure
+and/or existing `.xcconfig` files, you can reuse those, but probably need to adjust
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: but probably need => but you'll probably need

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 16]
```diff
@@ -0,0 +1,487 @@
...
+  MyApp/
+    MyApp/
+      AppDelegate.h
+      AppDelegate.m (or swift)
+      :
+```
+
+### Add your Flutter app to your Podfile
+
+Integrating the Flutter framework requires use of the CocoaPods dependency manager.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: requires use of the => requires using the

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 17]
```diff
@@ -0,0 +1,487 @@
...
+This is because the Flutter framework needs to be available also to any Flutter plugins
+that you might include in my_flutter.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: This is the first time you've mentioned "my_flutter". What is it referring to?

Delete "also".

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 18]
```diff
@@ -0,0 +1,487 @@
...
+
+Please refer to [cocoapods.org](https://cocoapods.org/) for how to install CocoaPods
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: => For information on how to install CocoaPods on your development machine,
see [cocoapods.org](xxx).

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 19]
```diff
@@ -0,0 +1,487 @@
...
+```
+
+3. Run `pod install`.
+
+Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
+you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
+of plugins read by the `podhelper.rb` script. Then run `pod install` again from
+`some/path/MyApp`.
+
+The `podhelper.rb` script will ensure that your plugins, the Flutter.framework, and the App.framework
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: will ensure => ensures

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 20]
```diff
@@ -0,0 +1,487 @@
...
+are embedded in your project.
+
+You should now be able to build the project using `⌘B`.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Is this on Mac only? Maybe also specify Windows?

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 21]
```diff
@@ -0,0 +1,487 @@
...
+
+### Under the hood
+
+If you have some reason to do this manually or debug why these steps aren't working, here's what's going on under the hood:
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Please use shorter lines. Also add a sentence:
"If you have some reason... Otherwise, skip this section."

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 22]
```diff
@@ -0,0 +1,487 @@
...
+
+1. `Flutter.framework` (the Engine library) is getting embedded into your app for you.  This has to match up with the release type
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Does this list need to be ordered? Can it be an unordered list? We only use an ordered list when the ordering matters.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 23]
```diff
@@ -0,0 +1,487 @@
...
+makes sure it gets embedded into your native app.
+3. Any plugins are added as CocoaPod pods.  In theory, it should be possible to manually merge those in as well, but those instructions
+vary on the pod dependencies of each plugin.
+4. A build script is added to the Podfile targets that call `install_all_flutter_pods` to ensure that the binaries you build stay up to date
+with the Dart code that's actually in the folder.  It also uses your Xcode build configuration (Debug, Profile, Release) to embed the matching
+release type of `Flutter.framework` and `App.framework`.
+
+## Write code to use FlutterViewController from your host app
+
+The proper place to do this will be specific to your host app. Here is an
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: "will be specific" => is specific

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 24]
```diff
@@ -0,0 +1,487 @@
...
+example that makes sense for the blank screen of the host app generated
+by Xcode.
+
+First declare your app delegate to be a subclass of `FlutterAppDelegate`. Then define a FlutterEngine property, which help you to register a plugin without a FlutterViewController instance.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Please break up this sentence.

", which help you to register" => ", which helps you register"

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 25]
```diff
@@ -0,0 +1,487 @@
...
+```objective-c
+#import <UIKit/UIKit.h>
+#import <Flutter/Flutter.h>
+
+@interface AppDelegate : FlutterAppDelegate
+@property (nonatomic,strong) FlutterEngine *flutterEngine;
+@end
+```
+
+This allows `AppDelegate.m` to be really simple, unless your host app
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Delete "really".

Also, "which needs to override other methods here:"  That's not quite clear. Can we reword? Even adding: "For example,..."

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 26]
```diff
@@ -0,0 +1,487 @@
...
+    return super.application(application, didFinishLaunchingWithOptions: launchOptions);
+  }
+
+}
+```
+
+<details>
+<summary>What to do if the app delegate already inherits from somewhere else.</summary>
+
+Make your app delegate implement the `FlutterAppLifeCycleProvider` protocol, e.g.:
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: We don't use "e.g.". Please replace with "for example" or "such as", etc.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 27]
```diff
@@ -0,0 +1,487 @@
...
+
+🔥  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
+An Observatory debugger and profiler on iPhone X is available at: http://127.0.0.1:54741/
+For a more detailed help message, press "h". To quit, press "q".
+```
+
+You can now edit the Dart code in `my_flutter`, and the changes can be hot
+reloaded by pressing `r` in the terminal. You can also paste the URL above into
+your browser to use the Dart Observatory for setting breakpoints, analyzing
+memory retention and other debugging tasks.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: OK, can this be accomplished using Dart DevTools? We've been asked NOT to highlight/use Observatory in our docs if it can be done with DevTools.

@jacob314

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 28]
```diff
@@ -0,0 +1,487 @@
...
+2. Run `flutter attach` with the `--isolate-filter` option.
+
+```bash
+$ flutter attach --isolate-filter='debug'
+Waiting for a connection from Flutter...
+Done.
+Syncing files to device...      1.1s
+
+🔥  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
+An Observatory debugger and profiler is available at: http://127.0.0.1:43343/
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Is this still the current message? I thought it had been replaced with a DevTools message.

@devoncarew

File: src/docs/development/add-to-app/ios/add-flutter-screen.md [hunk 29]
```diff
@@ -0,0 +1,487 @@
...
+For a more detailed help message, press "h". To detach, press "d"; to quit, press "q".
+
+Connected view:
+  debug isolate (isolates/642101161)
+```
+
+You can check out [`93573de`](https://github.com/flutter/flutter/commit/93573de2165c750fdeefcd2d620e2b8bd494fed6) for a more detailed example.
```
Comment (sfshaza2) [inline src/docs/development/add-to-app/ios/add-flutter-screen.md]: Please break up this sentence. Also:

You can check out [`93573de`]... => You can check out [commit 93573de]....

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 2 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 5 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 8 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 9 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 10 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 11 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 12 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: My concern with deleting it is that it is introducing the terminology "flutter module template" . I'm not sure if that is used in other documents.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 13 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 14 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 15 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 16 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 17 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: my_flutter is the running example, added tick marks

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 18 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 19 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 20 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: yes, mac is needed for ios

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 21 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: I'm not sure how to do that while preserving the colon =(

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 22 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 23 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 24 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 25 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 26 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: done

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 27 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: I'm not sure of the current status of DevTools.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 28 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Nope, it is still current.

File: src/docs/development/add-to-app/ios/add-flutter-screen.md (hunk 29 above)
Comment (gaaclarke) [reply src/docs/development/add-to-app/ios/add-flutter-screen.md]: Done.
